<?xml version="1.0"?>

<launch>
  <rosparam file="$(find base_controller)/config/params.yaml" command="load"/>
  <node pkg="base_controller" name="base_controller_node" type="hb_base_controller_node.py" output="screen"/>
</launch>
